HappyAnding/HappyAnding/TextLiteral.swift

```diff
@@ -0,0 +1,212 @@
+    static let writeShortcutTitleViewLinkPlaceholder: String = "단축어 링크를 입력하세요"
+    static let writeShortcutTitleViewOneLineTitle: String = "한 줄 설명"
+    static let writeShortcutTitleViewOneLinePlaceholder: String = "단축어의 핵심 기능을 입력하세요"
+    static let writeShortcutTitleViewMultiLineTitle: String = "상세 설명"
+    static let writeShortcutTitleViewMultiLinePlaceholder: String = "단축어 사용법, 필수적으로 요구되는 사항 등 단축어를 이용하기 위해 필요한 정보를 입력하세요"
+    static let writeShortcutTitleViewEdit: String = "단축어 편집"
+    static let writeShortcutTitleViewPost: String = "단축어 등록"
+    static let writeShortcutTitleViewCategoryTitle: String = "카테고리"
+    static let writeShortcutTitleViewCategoryDescription: String = "최대 3개"
+    static let writeShortcutTitleViewCategoryCell: String = "카테고리 선택"
+    static let writeShortcutTitleViewRequiredAppDescription: String = "(선택)"
+    static let writeShortcutTitleViewRequiredAppsTitle: String = "단축어 사용에 필요한 앱"
+    static let writeShortcutTitleViewRequiredAppInformation: String = "해당 단축어를 사용하기 위해 필수로 다운로드해야 하는 앱을 입력해주세요"
+    static let writeShortcutTitleViewRequiredAppCell: String = "앱 추가"
+    
+    // MARK: - IconModalView
+    static let iconModalViewTitle: String = "아이콘"
+    static let iconModalViewColor: String = "색상"
+    static let iconModalViewIcon: String = "기호"
+    
+    // MARK: - CategoryModalView
+    static let categoryModalViewTitle: String = "카테고리"
+    
+    // MARK: - ReadShortcutView
+    static let readShortcutViewBasicTabTitle: String = "기본 정보"
+    static let readShortcutViewVersionTabTitle: String = "버전 정보"
+    static let readShortcutViewCommentTabTitle: String = "댓글"
+    static let readShortcutViewDeletionTitle: String = "단축어 삭제"
+    static let readShortcutViewDeletionMessage: String = "단축어를 삭제하시겠어요?"
+    static let readShortcutViewDeleteFixes: String = "수정사항을 삭제하시겠어요?"
+    static let readShortcutViewKeepFixes: String = "계속 작성"
+    static let readShortcutViewCommentDescriptionBeforeLogin: String = "로그인 후 댓글을 작성할 수 있어요"
+    static let readShortcutViewCommentDescription: String = "댓글을 입력하세요"
+    
+    // MARK: - ReadShortcutContentView
+    static let readShortcutContentViewDescription: String = "단축어 설명"
+    static let readShortcutContentViewCategory: String = "카테고리"
+    static let readShortcutContentViewRequiredApps: String = "단축어 사용에 필요한 앱"
+    static let readShortcutContentViewRequirements: String = "단축어 사용을 위한 요구사항"
+    
+    // MARK: - ReadShortcutVersionView
+    static let readShortcutVersionViewNoUpdates: String = "업데이트된 버전이 없어요"
+    static let readShortcutVersionViewUpdateContent: String = "업데이트 내용"
+    static let readShortcutVersionViewDownloadPreviousVersion: String = "이전 버전 다운로드"
+    
+    // MARK: - ReadShortcutCommentView
+    static let readShortcutCommentViewNoComments: String = "등록된 댓글이 없어요"
+    static let readShortcutCommentViewDeletionTitle: String = "댓글 삭제"
+    static let readShortcutCommentViewDeletionMessage: String = "답글도 함께 삭제됩니다. 댓글을 삭제하시겠습니까?"
```

Comment:
```suggestion
    static let readShortcutCommentViewDeletionMessage: String = "답글도 함께 삭제돼요. 댓글을 삭제하시겠어요?"
```
이것도 바뀐 워딩에 들어있는 부분은 아닌데, 단축어 및 추천 모음집 삭제와 비슷한 어투로 변경하는 게 좋을 것 같습니다!